public/app/core/components/Tooltip/Popper.tsx

```diff
@@ -21,13 +21,21 @@ interface Props {
   placement?: any;
   content: string | ((props: any) => JSX.Element);
   refClassName?: string;
+  theme?: string;
+}
+
+export enum Themes {
+  Default = 'popper__background--default',
+  Error = 'popper__background--error',
 }
 
 class Popper extends PureComponent<Props> {
   render() {
-    const { children, renderContent, show, placement, refClassName } = this.props;
+    const { children, renderContent, show, placement, refClassName, theme } = this.props;
```

Comment:
@jschill please take a look at my changes in #14766. We will have to sync those.

Reply:
Definately 👍